examples/spin-timer/src/main.rs

```diff
@@ -103,9 +123,11 @@ impl TimerTrigger {
     async fn handle_timer_event(&self, component_id: &str) -> anyhow::Result<()> {
         // Load the guest...
         let (instance, mut store) = self.engine.prepare_instance(component_id).await?;
-        let engine = spin_timer::SpinTimer::new(&mut store, &instance, |data| data.as_mut())?;
+        let EitherInstance::Component(instance) = instance else {
```

Comment:
Does this mean the SQS trigger needs to be updated too?

Reply:
Yes.  If it's supposed to work with the main branch of Spin, we should update it now.  If it's only supposed to work with the latest stable release, we have until Spin v1.1 is released.  Either way, I'm happy to do it.

Reply:
We can defer until 1.1 for sure.

@lee-orr a heads up for you that this will affect https://github.com/lee-orr/spin-message-trigger as well (not immediately but just so you know it's coming)

Reply:
Thanks for the update!